greenplumpython/expr.py

```diff
@@ -24,13 +21,11 @@ class Expr:
 
     def __init__(
         self,
-        as_name: Optional[str] = None,
         table: Optional["Table"] = None,
         db: Optional[Database] = None,
     ) -> None:
-        self._as_name = as_name
         self._table = table
-        self._db = table.db if table is not None else db
+        self._db = db if db is not None else (table.db if table is not None else None)
```

Comment:
self._db = db or table.db?

Reply:
return None.
![image](https://user-images.githubusercontent.com/15976103/197717047-52a7ae6d-b15b-4683-9fff-c82629103ea9.png)

Reply:
get your point you are right.